x/oracle/keeper/validatorWhiteList.go

```diff
@@ -1,78 +1,108 @@
 package keeper
 
 import (
-	sdk "github.com/cosmos/cosmos-sdk/types"
-	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
+	"fmt"
 
 	"github.com/Sifchain/sifnode/x/oracle/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
-func (k Keeper) SetOracleWhiteList(ctx sdk.Context, validatorList []sdk.ValAddress) {
+// GetAllWhiteList set the validator list for a network.
```

Comment:
This comment makes it sound like we are doing a set action with this function when all this function does is fetch data.

Reply:
will udpate the coment.